lms/djangoapps/django_comment_client/utils.py

```diff
@@ -147,7 +147,7 @@ def get_discussion_id_map_entry(xblock):
         xblock.discussion_id,
         {
             "location": xblock.location,
-            "title": xblock.discussion_category.split("/")[-1].strip() + " / " + xblock.discussion_target
+            "title": xblock.discussion_category.split("/")[-1].strip() + (" / " + xblock.discussion_target if xblock.discussion_target else "")
```

Comment:
@noraiz-anwar `discussion_category` does not have the `discussion_target `.
I have checked this `xblock.discussion_category.split("/")[-2].strip()` and it does not exists.